### DISCUSSION TITLE
Export/Save/Print Discord DM/Chat IOS App and Windows website/app

### DISCUSSION THREAD

**Discussion by WonderWoman1977** [Tyrrrz/DiscordChatExporter#729] [Q&A]
Hello,

I am a tech, but my skills are in the repair not programming/scripting/coding area.  I have been asked if there is a way to save/print/export any sort of chat log/conversation/thread from Discord on various platforms including IOS, Mac and Windows.  I have been asked this by non-tech savvy people and I am wondering if there is any easy way to do this on any or all of those platforms?  All suggestions appreciated.  I have already DL'd the bot and plan on learning how to use it but I am a tech and the people asking me are not so I think that would be over their head.

All suggestions appreciated.

Sincerely,

WonderWoman1977

**Comment by 96-LB** [comment]
DiscordChatExporter can export channels as .HTML files, which you can view in any web browser. Viewing them on IOS might be a hassle, but they should be easily accessible on Mac and Windows.
